README.md

```diff
@@ -1,63 +1,172 @@
-# Axelar Local Dev Sample
+# Axelar cross-chain dapp examples
 
-This repo provides a basic usage of https://github.com/axelarnetwork/axelar-local-dev
+## Introduction
 
-## Install
+This repo provides the code for several example dapps in the [Axelar Local Development Environment](https://github.com/axelarnetwork/axelar-local-dev). It contains both the JavaScript and the smart contract code (.sol files) for each example. To try them out:
 
-1. Run `npm update && npm install`.
-2. Run `npm run build` to compile all contracts.
+- Set up your system.
+- Deploy and test the apps.
 
-## Examples
+Note: You may see example folders in this repo that are not described below. They are either placeholders for future apps, such as the `temp` folder, or they are dapps in progress and we'll add a description when they're finished.
 
-To run any example on your local machine first run `node scripts/createLocal` on a separate terminal.
+## Setup
 
-All examples use `0xBa86A5719722B02a5D5e388999C25f3333c7A9fb` as the user that deploys and runs all tests. This address is funded by default when creating a local set of chains. Make sure it is funded on testnets as well. Run `node/printBalances` (optionally specify an address if you wish to check an address other than the one above) to check the balances on all five supported testnets of any address.
+1. You'll need to have node.js installed to run network dapps. To make sure you have it installed, run `node -v`. If no version is returned, run
+`npm update && npm install`.
```

Comment:
`npm update && npm install` must be run after you clone the repo, not before.

ensuring correct version of node is a separate one-time step.  Suggestion: section "one-time setup" do clone, check node version, `npm update && npm install`, `npm run build`